File: Interpreter/ConstManager.java


import java.util.Set
import java.util.TreeMap

class ConstManager {
	TreeMap<String, Complex> constMap = new TreeMap<String, Complex>()
	
	void registerConst(String key, Complex value){
		constMap.put(key, value)
	}
	
	Set<String> consts(){
		return constMap.keySet()
	}
	
	Complex find(String key){
		return constMap.get(key)
	}
	
	int getConstCount(){
		return constMap.size()
	}
}

File: Interpreter/ConstManager.py
class ConstManager:
	def __init__(self):
		self.constMap = dict()
	
	def registerConst(self, key, value):
		self.constMap[key] = value

	def consts(self):
		return self.constMap.keys()

	def find(self, key):
		return self.constMap.get(key)

	def getConstCount(self):
		return len(self.constMap.keys())
